torchao/quantization/prototype/mixed_precision/scripts/sensitivity_study.py

```diff
@@ -0,0 +1,95 @@
+import torch
+import torch.nn as nn
+
+from transformers import AutoModelForCausalLM, AutoTokenizer
+
+from lm_eval.models.huggingface import HFLM
+from lm_eval.evaluator import evaluate
+from lm_eval.tasks import get_task_dict
+
+from torchao.quantization import quantize_, int8_weight_only, int4_weight_only, int8_dynamic_activation_int4_weight
+from torchao._models._eval import TransformerEvalWrapper
+
+from torchao.quantization.quant_primitives import (
+    MappingType,
+    ZeroPointDomain,
+)
+
+from torchao.quantization.quant_api import (
+    change_linear_weights_to_int4_woqtensors,
+    change_linear_weights_to_int8_dqtensors,
+    change_linear_weights_to_int8_woqtensors,
+    autoquant,
+)
+
+torch._inductor.config.force_fuse_int_mm_with_mul = True
+torch._inductor.config.fx_graph_cache = True
+
+def intN_weight_only(group_size=32, n=8):
+    def apply_intN_weight_only_quant(weight):
+        # avoid circular dep
+        from torchao.dtypes import to_affine_quantized
+
+        mapping_type = MappingType.ASYMMETRIC
+        block_size = (1, group_size)
+        target_dtype = torch.uint8
+        quant_min = 0
+        quant_max = 2**n-1
+        eps = 1e-6
+        preserve_zero = False
+        zero_point_dtype = torch.bfloat16
+        zero_point_domain = ZeroPointDomain.FLOAT
```

Comment:
why is this `FLOAT`?

Reply:
In asymmetric quant, I found preserve_zero=True+ZeroPointDomain.INT has lower sqnr and worse accuracy than   preserve_zero=False+ZeroPointDomain.Float.

Reply:
Thanks for pointing it out. Just changed it to INT.